src/AggregateFunctions/AggregateFunctionMLMethod.cpp

```diff
@@ -104,8 +104,229 @@ namespace
 
 void registerAggregateFunctionMLMethod(AggregateFunctionFactory & factory)
 {
-    factory.registerFunction("stochasticLinearRegression", createAggregateFunctionMLMethod<FuncLinearRegression>);
-    factory.registerFunction("stochasticLogisticRegression", createAggregateFunctionMLMethod<FuncLogisticRegression>);
+    // stochasticLinearRegression documentation
+    FunctionDocumentation::Description description_linear = R"(
+This function implements stochastic linear regression.
+It supports custom parameters for:
+- learning rate
+- L2 regularization coefficient
+- mini-batch size
+
+It also has a few methods for updating weights:
+- Adam (used by default)
+- simple SGD
+- Momentum
+- Nesterov
+
+**Usage**
+
+The function is used in two steps: fitting the model and predicting on new data.
+
+1. Fitting
+
+For fitting a query like this can be used:
+
+```sql
+CREATE TABLE IF NOT EXISTS train_data
+(
+    param1 Float64,
+    param2 Float64,
+    target Float64
+) ENGINE = Memory;
+
+CREATE TABLE your_model ENGINE = Memory AS SELECT
+stochasticLinearRegressionState(0.1, 0.0, 5, 'SGD')(target, x1, x2)
+AS state FROM train_data;
+```
+
+Here, we also need to insert data into the `train_data` table.
+The number of parameters is not fixed, it depends only on the number of arguments passed into `linearRegressionState`.
```

Comment:
"The number of parameters is not fixed" refers to l. 110 - 113, right? The reader is confused, there are just three parameters: learning rate, L2 regularization coefficient and mini-batch size.